docs/design.md

```diff
@@ -4,8 +4,7 @@ title: GPJax Principles
 
 GPJax is designed to be a Gaussian process package that provides an
 accurate representation of the underlying maths. Variable names are
-designed to closely, if not exactly, match the notation in the
-:cite[rasmussen2006gaussian]{.title-ref}. We here list the notation used
+designed to closely, if not exactly, match the notation in the <strong data-cite="rasmussen2006gaussian"></strong>. We here list the notation used
```

Comment:
This is still broken.

Also, `if not exactly, match the notation in the`, drop `the`.